spec/parser_spec.lua

```diff
@@ -398,12 +409,6 @@ describe("Parser /", function()
             })
         end)
 
-        it("cannot be followed by a multiple semicolons", function()
-            assert_statements_error([[
-                return;;
-            ]], "expected 'end' before ';', to close the 'function' at line 2")
-        end)
```

Comment:
We should not have deleted this test case. This is intentional, because that is what Lua does. As I said before, semicolons after the return statement are an exception...

Reply:
My bad, I thought you meant it the other way around.